components/TheFooter.vue

```diff
@@ -26,12 +26,12 @@
                 </div>
 
                 <div class="px-5 py-2">
-                    <a
-                        href="https://docs.google.com/document/d/1i6oXabir-628csa2bQacHpc17Kmhec9EfkyHij9yQGM/edit#heading=h.sns5qzyi4gru"
+                    <NuxtLink
```

Comment:
hmmmmmm I wonder if the build wouldn't crawl these if we changed them back to an a tag. 


NuxtLink probably thinks this is an internal page and then freaks out when really it's external

Reply:
so I need to get this back to a href, right?

Reply:
I think so, I could be wrong